mypy.ini

```diff
@@ -75,3 +75,6 @@ ignore_missing_imports = True
 
 [mypy-example_interfaces.*]
 ignore_missing_imports = True
+
+[mypy-ros2bag.*]
```

Comment:
New dependency - should be added here https://github.com/5G-ERA/relay_network_application/blob/ros2/.github/workflows/pants.yaml#L29? Maybe not, if it it in the base installation of ROS.

Reply:
according to the official documentation it is part of any ros-base installation 

> rosbag2 packages are available via debian packages, and will already be included with any -ros-base installation